packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -21,11 +21,12 @@ ${chalk.dim('Options:')}
 
   -h, --help                                                        Show usage information
       --type    allocate|unallocate|reallocate|collect              Filter by type
-      --status  queued|approved|pending|success|failed|canceled     Filter by status 
+      --status  queued|approved|pending|success|failed|canceled     Filter by status
       --source <source>                                             Fetch only actions queued by a specific source
       --reason <reason>                                             Fetch only actions queued for a specific reason
       --orderBy id|deploymentID|amount|priority|...|updatedAt       Order actions by a specific field (default: id)
       --orderDirection asc|desc                                     Order direction (default: desc)
+      --first [N]                                                   Fetch only the N first records (defaults to 1, returns all records if omitted)
```

Comment:
What do you think about defaulting to 100? That way we don't cause significant changes in the current default behavior.